modules/@apostrophecms/schema/ui/apos/components/AposInputJoin.vue

```diff
@@ -24,7 +25,11 @@
           />
         </div>
         <AposSlatList @update="updated" :initial-items="items" />
-        <AposSearchList :list="searchList" @select="selected" :selected-items="items" />
+        <AposSearchList
+          :list="searchList"
+          @select="selected"
```

Comment:
The selection of searched items (to include them in the SlatList) is not working anymore